source/develop/developer-guide/vdsm/developers.html.md

```diff
@@ -41,7 +41,13 @@ Install the following packages before you attempt to build VDSM:
        python-ethtool m2crypto python-dmidecode python-netaddr python-inotify python-argparse git \
        python-cpopen bridge-utils libguestfs-tools-c pyparted openssl libnl3 libtool gettext-devel python-ioprocess \
        policycoreutils-python python-simplejson python-blivet python-six mom ovirt-vmconsole
-
+       
+Install Python3 packages (for fedora)
```

Comment:
add a new line here. check how this change is rendered with the view button.
current text needs formatting fixes.